Dockerfile

```diff
@@ -1,16 +1,16 @@
 # Dockerfile for ZooKeeper
 
-FROM quay.io/signalfuse/maestro-base:alp-3.8-jdk8
+FROM quay.io/signalfuse/signalfx-base:alp-3.8-jdk8
```

Comment:
Is this change intentional? That brings much more than just fixing the link to the ZooKeeper tarball.

Reply:
Yes it is intentional and afaict required for the image to work within the GKE realms

Reply:
In this case it'd be preferable for those two changes to be in distinct commits and your PR to call out both changes specifically.